src/main/java/in/projecteka/datanotificationsubscription/subscription/model/SubscriptionRepresentation.java

```diff
@@ -0,0 +1,27 @@
+package in.projecteka.datanotificationsubscription.subscription.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+public class SubscriptionRepresentation {
```

Comment:
Let's rename this to `SubscriptionRequestRepresentation`. We might have two different entities going forward (`SubscriptionRequest` and `Subscription`). Let's not mix both.

Reply:
The `SubscriptionRequest` is the Request coming from HIU, and `Subscription` could be individual subscription if we want to split based on `Category` or HIP basis.